beetsplug/bpd/__init__.py

```diff
@@ -72,6 +73,13 @@
     u'close', u'commands', u'notcommands', u'password', u'ping',
 )
 
+SUBSYSTEMS = [
+    u'update', u'player', u'mixer', u'options', u'playlist', u'database',
+    # Related to unsupported commands:
+    # u'stored_playlist', u'output', u'subscription', u'sticker', u'message',
+    # u'partition',
+]
```

Comment:
A quick comment here to say that subsystems are used for MPD's `idle` command could be helpful.